grants_tagger/scispacy_meshtagger.py

```diff
@@ -0,0 +1,59 @@
+"""
+Wrapper for scispacy mesh tagger. SciSpacy implements mesh
+tagging as a named entity recognition and linking step so
+the class also transforms the problem to multi label classification
+"""
+from collections import defaultdict
+
+from scispacy.linking import EntityLinker
+from scispacy.abbreviation import AbbreviationDetector
+from sklearn.metrics import f1_score
+from scipy.sparse import csr_matrix
+import scispacy
+import spacy
+
+
+class SciSpacyMeshTagger():
+    def __init__(self, mesh_tags, threshold=2):
+        """
+        mesh_tags: list of mesh UI descriptros e.g. [D00001]
+        """
```

Comment:
could do with a description of what `threshold` is

Reply:
99d9c7e843ce7696eb62e9b7132f5d66cf7d3d37